pkg/executor/executor.go

```diff
@@ -270,6 +270,16 @@ func StartExecutor(ctx context.Context, logger *zap.Logger, functionNamespace st
 
 	executorInstanceID := strings.ToLower(uniuri.NewLen(8))
 
+	namespace, err := utils.GetCurrentNamespace()
+	if err != nil {
+		logger.Warn("Current namespace not found %v", zap.Error(err))
+	}
+
+	podSpecPatch, err := util.GetSpecFromConfigMap(ctx, kubernetesClient, fv1.RuntimePodSpecConfigmap, namespace)
```

Comment:
we are still executing this function even if error is returned by GetCurrentNamespace.

If GetCurrentNamespace returns error we should not execut this function.